conf/default-vars.yaml

```diff
@@ -12,7 +12,6 @@ stylesheets:
         regular: 400
         semibold: 600
```

Comment:
I feel like we should never have two different bold values defined.  I don't think we'll ever have a site that truly uses 4 weights - probably just 3

Reply:
Ok, we can always add it back.